tests/hawc/apps/epiv2/test_api.py

```diff
@@ -0,0 +1,201 @@
+import pytest
+from django.core.exceptions import ObjectDoesNotExist
+from django.urls import reverse
+from rest_framework.test import APIClient
+
+from hawc.apps.assessment.models import Log
+from hawc.apps.epiv2 import models
+
+
+@pytest.mark.django_db
+class TestDesignApi:
+    def test_permissions(self, db_keys):
+        url = reverse("epiv2:api:design-list")
+        data = {
+            "study": db_keys.study_working,
+            "study_design": "CC",
+            "source": "GP",
+            "age_description": "10-15 yrs",
+            "sex": "B",
+            "race": "Not specified but likely primarily Asian",
+            "summary": "Case-control study of asthma in children",
+            "study_name": "Genetic and Biomarkers study for Childhood Asthma",
+            "region": "northern Taiwan",
+            "years": "2009-2010",
+            "participant_n": 456,
+            "age_profile": ["Children and adolescents <18 yrs"],
+            "countries": ["TW"],
+        }
+        generic_perm_tester(url, data)
+
+    def test_valid_requests(self, db_keys):
+        url = reverse("epiv2:api:design-list")
+        client = APIClient()
+        assert client.login(username="team@hawcproject.org", password="pw") is True
+
+        data = {
+            "study": db_keys.study_working,
+            "study_design": "CC",
+            "source": "GP",
+            "age_description": "10-15 yrs",
+            "sex": "B",
+            "race": "Not specified but likely primarily Asian",
+            "summary": "test epidemiology study design",
+            "study_name": "Genetic and Biomarkers study for Childhood Asthma",
+            "region": "northern Taiwan",
+            "years": "2009-2010",
+            "participant_n": 456,
+            "age_profile": ["Children and adolescents <18 yrs"],
+            "countries": ["TW"],
+        }
+
+        just_created_design_id = None
```

Comment:
this is a strange pattern? should this be a fixture or a setup method instead?

Reply:
Just curious, what strategy did you end up going with and why? This'll be useful for documentation...

Reply:
I decided to postpone making changes to the tests, we can make a card to remove this pattern on both epi version 1 and 2

Reply:
That makes sense!